src/site/content/en/blog/storage-foundation/index.md

```diff
@@ -0,0 +1,372 @@
+specific use-cases in mind.
+
+- Some of these options clearly do not overlap with this proposal as they only allow very small
+  amounts of data to be stored, like
+  [cookies](https://developer.mozilla.org/en-US/docs/Web/HTTP/Cookies), or the
+  [Web Storage API](https://developer.mozilla.org/en-US/docs/Web/API/Web_Storage_API) consisting of
+  the `sessionStorage` and the `localStorage` mechanisms.
+- Other options are already deprecated for various reasons like the
+  [File and Directory Entries API](https://developer.mozilla.org/en-US/docs/Web/API/File_and_Directory_Entries_API/Introduction)
+  or [WebSQL](https://www.w3.org/TR/webdatabase/).
+- The [File System Access API](/file-system-access/) has a similar API surface, but its main
+  intended usage is to interface with the client's filesystem and provide access to data that may be
+  outside of the origin's or even the browser's ownership. This different focus comes with stricter
+  security considerations and higher performance costs.
+- The [IndexedDB API](https://developer.mozilla.org/en-US/docs/Web/API/IndexedDB_API) can be used as
+  a backend for some of the Storage Foundation API's use-cases. For example, Emscripten includes
+  [IDBFS](https://emscripten.org/docs/api_reference/Filesystem-API.html), an IndexedDB-based
+  persistent file system. However, since IndexedDB is fundamentally a key-value store, it comes with
+  significant performance limitations. Furthermore, directly accessing sub-sections of a file is
+  even more difficult and slower under IndexedDB.
+- Finally, the
+  [CacheStorage interface](https://developer.mozilla.org/en-US/docs/Web/API/CacheStorage) is widely
+  supported and is tuned for storing large-sized data such as web application resources, but the
+  values are immutable.
+
+The Storage Foundation API is an attempt at closing all the gaps of the previous storage options by
+allowing for the performant storage of mutable large files defined within the origin of the
+application.
+
+### Suggested use cases for the Storage Foundation API {: #use-cases }
+
+Examples of sites that may use this API include:
+
+- Productivity or creativity apps that operate on large amounts of video, audio, or image data. Such
+  apps can offload segments to disk instead of holding them in memory.
+- Apps that rely on a persistent filesystem accessible from Wasm and that need more performance than
+  what IDBFS can guarantee.
+
+## What is the Storage Foundation API? {: #what }
+
+There are two main parts to the API:
+
+- **File system calls**, which provide basic functionality to interact with files and file paths.
+- **File handles**, which provide read and write access to an existing file.
+
+### File system calls
+
+The Storage Foundation API introduces a new interface `storageFoundation` that lives on the `window`
+object and that includes a number of functions:
+
+{% Aside %} We are currently exploring the tradeoffs between providing a synchronous vs. an
```

Comment:
I don't think there is a way, but the good thing is that this Aside will eventually go away since it's temporary. There are one or two other such temporary Asides…